schemas/src/registry_schemas/schemas/name_request.json

```diff
@@ -0,0 +1,52 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request",
+    "type": "object",
+    "title": "Name Request Schema",
+    "properties": {
+        "nrNumber": {
+            "type": "string"
```

Comment:
We can lock this down to a 10-character string, since that's what it is. Also starts with "NR", ie: "NR 1234567".

Reply:
ok great, will do!